src/ext/state_context.rs

```diff
@@ -2,7 +2,8 @@ use crate::{cstr, plugin::*, stream::*};
 
 use std::ffi::CStr;
 
-pub const CLAP_EXT_STATE_CONTEXT: &CStr = cstr!("clap.state-context.draft/1");
+pub const CLAP_EXT_STATE_CONTEXT: &CStr = cstr!("clap.state-context/1");
+pub const CLAP_EXT_STATE_CONTEXT_COMPAT: &CStr = cstr!("clap.state-context.draft/1");
 
 pub const CLAP_STATE_CONTEXT_FOR_DUPLICATE: clap_plugin_state_context_type = 1;
```

Comment:
Missing some changes to these constants: https://github.com/free-audio/clap/compare/1.1.10..1.2.0#diff-b95d5c5682aecd38a3e93eff208f59826ae0ad26e7c043b232d07dd4f817c869R38-R45